src/screens/HomePage/HomePage.js

```diff
@@ -29,12 +39,29 @@ export default function HomePage() {
   const shareBlockRef = useRef(null);
   const location = useLocation();
 
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('xs'));
```

Comment:
Also here, `theme.breakpoints.down('sm')`

Reply:
same as above, this seems to be working as intended